.github/mergify.yml

```diff
@@ -118,6 +118,14 @@ pull_request_rules:
   actions:
     merge:
       method: merge
+      commit_message_template: |
+        {{ title }} (#{{ number }})
+
+        {{ body }}
+
+        {% for user in approved_reviews_by %}
```

Comment:
I am ok with the title and body. To me, the approved-by is just noise in the git log. If you need that level of information for a commit, you can find it in the PR.

Reply:
GitHub already allows configuring the title and body for merge commits [natively](https://github.blog/changelog/2022-08-23-new-options-for-controlling-the-default-commit-message-when-merging-a-pull-request/?ref=blog.mergify.com). However, this PR is focused on adding an `Approved-By` field. This information isn't readily available by just looking at the git history, but it’s crucial for quickly identifying who authored and approved a PR. Including this field enhances accountability and traceability, making it easier to see who was responsible for key decisions.

Reply:
> This information isn't readily available by just looking at the git history

Yes but the values in the field are GitHub ids and not something general git like committer/author emails. So seeing a GitHub id in the commit message means you still need to go to GitHub to map the id back to someone. I would need to go to GitHub to "decode" `leseb` to some person.

Reply:
I fixed this issue in my latest push.